### PR TITLE
Aside anchors

### PR DESCRIPTION
**Pull request by vinicios-gomes**
Fixed insertion of last youtube video

### CODE REVIEW

**Review by liviacarolgouvea — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  // Use o IntelliSense para saber mais sobre os atributos possíveis.
+  // Focalizar para exibir as descrições dos atributos existentes.
+  // Para obter mais informações, acesse: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
```
Comment (vitormattos) [inline .vscode/launch.json:5]: Don't version `.vscode` folder. If necessary, put in `.gitignore` file

File: js/index.js [hunk 2]
```diff
@@ -49,9 +72,10 @@ if (listGroup) {
 		}
 	}
 }
-
 const orcidLabel = document.getElementsByClassName('label');
 const orcidID = document.getElementsByClassName('orcid');
+const videoId = getDataByGoogle('key', 'UCBc00LEzZ6Vqi8EcBoCLaYA');
```
Comment (vitormattos) [inline js/index.js:77]: does this value have to be hardcode?

File: js/index.js [hunk 3]
```diff
@@ -49,9 +72,10 @@ if (listGroup) {
...
+inserVideo(videoId, 'localhost');
```
Comment (vitormattos) [inline js/index.js:78]: value hardcoded?

File: .vscode/launch.json (hunk 1 above)
Comment (vinicios-gomes) [reply .vscode/launch.json:5]: Solved in [Commit](https://github.com/LyseonTech/ojs-csp-theme/pull/20/commits/b9cd600acc6bda93351361b3609109a588d82bf7)

File: js/index.js (hunk 2 above)
Comment (vinicios-gomes) [reply js/index.js:77]: Within the OJS there is no specific field to inform this data.
What do you suggest to be a better alternative?

File: js/index.js (hunk 3 above)
Comment (vinicios-gomes) [reply js/index.js:78]: Within the OJS there is no specific field to inform this data.
What do you suggest to be a better alternative?

File: js/index.js (hunk 2 above)
Comment (vinicios-gomes) [reply js/index.js:77]: **Duplicated**.

**Pull request merged**